package.json

```diff
@@ -35,7 +35,7 @@
   },
   "homepage": "https://github.com/silverstripe/silverstripe-framework#readme",
   "dependencies": {
-    "babel-polyfill": "^6.7.4",
+    "babel-polyfill": "https://registry.npmjs.org/babel-polyfill/-/babel-polyfill-6.7.4.tgz",
```

Comment:
It seems to be related to running `npm install <package>` when it's already present in `npm-shrinkwrap.json`. We haven't tracked it down yet. I've manually fixed it for now.